services/coderabbit/coderabbit-stats.service.js

```diff
@@ -0,0 +1,69 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  reviews: Joi.number().required(),
+}).required()
+
+class CodeRabbitStats extends BaseJsonService {
+  static category = 'analysis'
+  static route = {
+    base: 'coderabbit',
+    pattern: 'stats/:provider/:org/:repo',
+  }
+
+  static examples = [
+    {
+      title: 'CodeRabbit Review Stats',
+      namedParams: {
+        provider: 'github',
+        org: 'coderabbitai',
+        repo: 'ast-grep-essentials',
+      },
+      staticPreview: this.render({
+        reviews: 101,
+      }),
+      documentation: 'Shows the number of CodeRabbit reviews for a repository',
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'CodeRabbit',
+    labelColor: '171717',
+  }
+
+  static render({ reviews }) {
+    return {
+      message: `${reviews} Reviews`,
+      color: 'ff570a',
+    }
+  }
+
+  static renderError({ message }) {
+    return {
+      message,
+      color: '9f9f9f',
+    }
+  }
+
+  async fetch({ provider, org, repo }) {
+    return this._requestJson({
+      schema,
+      url: `https://api.coderabbit.ai/stats/${provider}/${org}/${repo}`,
+      httpErrors: {
+        404: 'invalid',
```

Comment:
Any particular reason not to stick with the default 404 message ("not found") here? If we want to override it, something more descriptive (e.g: "project not found", "repo not found" etc) would be more helpful than "invalid"

Reply:
I have made the error, repo not found. Updated tests.